dom.bs

```diff
@@ -1196,9 +1196,13 @@ method, when invoked, must run these steps:
  <a>report a warning to the console</a> that this might not give the expected results.
  [[!SERVICE-WORKERS]]
 
- <li><p>Set <var>listener</var>'s <a for="event listener">removed</a> to true and
- <a for=list>remove</a> <var>listener</var> from <var>eventTarget</var>'s
- <a for=EventTarget>event listener list</a>.
+ <li><p>If <var>eventTarget</var>'s <a for=EventTarget>event listener list</a> does not
+ <a for=list>contain</a> an <a>event listener</a> whose <a for="event listener">type</a> is
+ <var>listener</var>'s <a for="event listener">type</a>, <a for="event listener">callback</a> is
+ <var>listener</var>'s <a for="event listener">callback</a>, and <a for="event listener">capture</a>
+ is <var>listener</var>'s <a for="event listener">capture</a>, then set <var>listener</var>'s
+ <a for="event listener">removed</a> to true and <a for=list>remove</a> <var>listener</var> from
+ <var>eventTarget</var>'s <a for=EventTarget>event listener list</a>.
 </ol>
 
 <p class=note>HTML needs this to define event handlers. [[HTML]]
```

Comment:
Maybe update this note to indicate that the contains check above always no-ops when HTML calls this? Or maybe call that out in an HTML comment. Just keep that knowledge around somewhere.

Reply:
Appended this note in a203af5 instead of HTML spec, so we can refer `contains` check more easily (as "above").

I was a bit hesitant to use "check above always no-ops" wording, as it may be understood as "listener is never found, so step 3 does nothing" and went with "always returns true".